scripts/TEST.vcf

```diff
@@ -0,0 +1,500 @@
+##fileformat=VCFv4.2
+##FILTER=<ID=PASS,Description="All filters passed">
+##FILTER=<ID=Consensus,Description="Called by two or more callers">
+##FILTER=<ID=Hotspot,Description="In mutational hotspot, as defined by ????">
+##INFO=<ID=MQ,Number=1,Type=Integer,Description="RMS mapping quality (normal sample)">
+##INFO=<ID=MQ0,Number=1,Type=Integer,Description="Number of MAPQ=0 reads (normal sample)">
+##INFO=<ID=CAL,Number=.,Type=String,Description="List of callers making this call">
+##FORMAT=<ID=AGT,Number=.,Type=String,Description="Consensus or majority genotype">
```

Comment:
Hmmm, we can discuss this a bit, but I'd like to keep the ID as is in the reserved style spec format. See page 8-11 [here](https://samtools.github.io/hts-specs/VCFv4.3.pdf). That will allow for compatibility with downstream tools. The description can be updated though to reflect the unorthodox nature of how the value was arrived at

Reply:
That is, you'd like this 'consensus genotype' to continue to be called 'GT', rather than 'AGT' as here? That could be ok, the main reason I created this different tag was to permit the use of the value 'deadlock' or whatever it was instead of a real genotype if multiple genotypes had equal support among the callers. Some ticket or discussion had suggested assigning '0/1' in that case instead. I don't feel amazing about that (consider random assignment or most reference-y GT?), but if that's still the preferred solution, I am ok with it.

Reply:
You're assessment is correct. Would call it a "reluctant preference" in that some downstream tools/DBs "want" to see a traditional `GT` field. So, the hope is that we can explain that in that description (that we use consensus of callers, but use `0/` when no consensus exists), and that the presence of the `GT_STATUS` and per-caller GT values would help. For the record, I also am not jazzed about using `0/1` as default, but the rigidity of the field spec ties our hands. Besides, `GT` is somatic calling is pretty much a joke anyway, right? 😉

Reply:
Ok, I created this with 0/1 as the genotype of last resort.